### ISSUE TITLE
Open with SteamVR

### ISSUE THREAD

**Issue by designeerlabs** [designeerlabs/discord-vr#7]
Add the ability to launch with SteamVR as an "addon"

**Comment by BrianGilbert** [comment]
Some resources that may help when you get to this:

https://github.com/ValveSoftware/openvr/tree/master/samples/helloworldoverlay

https://github.com/sh-akira/VROverlay

Desktop app + Overlay: https://github.com/pushrax/OpenVR-SpaceCalibrator

Advanced Settings: https://github.com/OpenVR-Advanced-Settings/OpenVR-AdvancedSettings should be main.cpp and overlaycontroller.cpp is all you are really interested in.

Input Emulator: its a driver + overlay app https://github.com/matzman666/OpenVR-InputEmulator

**Comment by burritosoftware** [comment]
It's very easy to do this by making a vrmanifest (basically just a json file). Heck, I might just do it right now since the app dev doesn't have to do it themselves.

**Comment by burritosoftware** [comment]
Here's a vrmanifest for launching on startup. (I had to wrap it in a zip so I can upload it to GitHub.)

Put the .vrmanifest file in the same place where the DiscordVR executable is. Then, open `C:\Program Files (x86)\Steam\config\appconfig.json` (or wherever you installed Steam, then config then appconfig.json)

Add a new JSON value and enter in the path to your vrmanifest. Make sure to escape slashes like all the other paths.

[app.zip](https://github.com/designeerlabs/discord-vr/files/6527454/app.zip)

**Comment by designeerlabs** [comment]
Yep I got it working last week, thanks though guys. I'll make a branch with the updates I'm making.
